pages/_app.js

```diff
@@ -106,9 +118,17 @@ export default function App({ Component, pageProps }) {
     setUseExampleDate(!useExampleData);
   }
 
+  // function for StartModal
+
+  function handleDemoMode() {
+    setDemoMode(true);
+    setUseExampleDate(true);
+  }
+
   const {
     data: dbEmotionEntries,
     isLoading,
+    error,
```

Comment:
I think you don't use this error here, or? You could therefore remove it again.

Reply:
because of the merge we need it now